src/main/java/uk/gov/hmcts/cmc/claimstore/services/ClaimService.java

```diff
@@ -182,4 +182,9 @@ public void saveDefendantResponse(final long claimId, final String defendantId,
                                final ResponseData responseData) {
         claimRepository.saveDefendantResponse(claimId, defendantId, defendantEmail, jsonMapper.toJson(responseData));
     }
+
+    public void linkSealedClaimDocumentManagementPath(final Long claimId,
+                                                      final String sealedClaimDocumentManagementSelfPath) {
```

Comment:
I would rename arg name here as well to be something shorter then `sealedClaimDocumentManagementSelfPath`. A `documentSelfPath` should be enough IMO.

Reply:
done